addons/mail/models/mail_channel.py

```diff
@@ -52,6 +54,7 @@ def _get_default_image(self):
     channel_last_seen_partner_ids = fields.One2many(
         'mail.channel.partner', 'channel_id', string='Last Seen',
         groups='base.group_user')
+    rtc_sessions = fields.One2many('mail.rtc.session', 'channel_id')
```

Comment:
Don't add fields randomly in fields definition. Purpose would be to keep a logical ordering  in definition.